src/Administration/Resources/app/administration/src/module/sw-settings-document/snippet/de-DE.json

```
@@ -69,7 +74,8 @@
       "labelDisplayDivergentDeliveryAddress": "Abweichende Lieferadresse anzeigen",
       "helpTextDisplayDocumentInCustomerAccount": "Das Dokument wird dem Shopkunden unter Bestellungen im Kundenkonto angezeigt. Nur Dokumente die an den Kunden gesendet wurden, werden im Kundenkonto angezeigt.",
       "helpTextDisplayDeliveryCountries": "Alle ausgewählten Länder erhalten einen Hinweis zur \"innergemeinschaftlichen Lieferung\" auf den Rechnungsdokumenten.",
-      "helpTextCompanyAddress": "Verwende Bindestriche (\"-\") um die einzelnen Elemente der Adresse voneinander abzugrenzen. Das hilft bei der gleichzeitigen Darstellung der Adresse an unterschiedlichen Stellen der Kopfzeile."
+      "helpTextCompanyAddress": "Veraltet für 6.7 - Benutze die neuen einzelnen Felder für die Adresse. Verwende Bindestriche (\"-\") um die einzelnen Elemente der Adresse voneinander abzugrenzen. Das hilft bei der gleichzeitigen Darstellung der Adresse an unterschiedlichen Stellen der Kopfzeile.",
```

same here: "obsolet ab Shopware 6.7 - ..."

---

It is also difficult to distinguish between the deprecation and the normal hint. Maybe:
```suggestion
      "helpTextCompanyAddress": "Obsolet ab 6.7. Fülle die Adresse stattdessen in die einzelnen Felder.\n\nVerwende Bindestriche (\"-\") um die einzelnen Elemente der Adresse voneinander abzugrenzen. Das hilft bei der gleichzeitigen Darstellung der Adresse an unterschiedlichen Stellen der Kopfzeile.",
```